wiki/Tournaments/OMIC/2019_4K/id.md

```diff
@@ -0,0 +1,320 @@
+---
+tags:
+- O!MIC 4K 2019
+- OMIC
+---
+# osu!mania 4K Indonesia Cup 2019
+![O!MIC 4K 2019 Logo](img/logo.png)
+**osu!mania 4K Indonesia Cup 2019** adalah turnamen osu!mania 4K tunggal khusus wilayah Indonesia dengan memakai sistem pertandingan *Royal Rumble* pada babak setelah Kualifikasi, lalu satu lawan satu dengan *Double-Elimination bracket* dimulai dari Penyisihan Grup.
```

Comment:
the tournament's acronym should be written in bold italics near the title. idk what works best for indonesian, but in english we put it in parenthesis after the title